src/transformStateWithClones.js

```diff
@@ -7,7 +7,42 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let prevState = state;
+
+  for (const action of actions) {
+    let newState = transform (prevState, action);
```

Comment:
why let here? you don't change it anywhere
```suggestion
    const newState = transform (prevState, action);
```